### ISSUE TITLE
buttons only occasionally work

### ISSUE THREAD

**Issue by cawong89** [Exslims/MercuryTrade#288]
I just installed the latest version of MercuryTrade. It appears to work, but only sometimes. For example, when I get a whisper, the "+" button to invite a player only seems to work about half the time. Often I need to click a button multiple times before it works. Furthermore, sometimes the "kick player" button closes the notification without successfully kicking the player, but not all the time. Similarly, some trade whispers occasionally seem to not cause notifications to appear.

In short, all the basic functions of MercuryTrade appear to work, but only with about 50% success rate. Any known workarounds?
